clrdefinitions.cmake

```diff
@@ -7,7 +7,7 @@ endif()
 if (WIN32)
   set(FEATURE_EVENT_TRACE 1)
 endif()
-if(CLR_CMAKE_PLATFORM_LINUX AND CLR_CMAKE_PLATFORM_ARCH_AMD64)
+if(CLR_CMAKE_PLATFORM_LINUX AND CLR_CMAKE_TARGET_ARCH_AMD64)
   set(FEATURE_EVENT_TRACE 1)
```

Comment:
should this also get enabled for target_arch_arm64? And why it was not enabled for other arch (arm, x86)